<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fct_part3_2</class>
 <widget class="QWidget" name="fct_part3_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Suppression</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ajout</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Modification</string>
   </property>
  </widget>
  <widget class="QComboBox" name="numEpAjout">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="orAjout">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="argentAjout">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>120</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="bronzeAjout">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>120</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>-10</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Gestion des résultats</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ajouterButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modifierBouton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QPushButton" name="supprimerBouton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QComboBox" name="numEpModif">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="numEpSupp">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Numéro épreuve</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Médaille d'or</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Médaille d'argent</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Médaille de bronze</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_fct_part3_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>631</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ajouterButton</sender>
   <signal>clicked()</signal>
   <receiver>fct_part3_2</receiver>
   <slot>ajouterRes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>modifierBouton</sender>
   <signal>clicked()</signal>
   <receiver>fct_part3_2</receiver>
   <slot>ouvrirModifierRes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>supprimerBouton</sender>
   <signal>clicked()</signal>
   <receiver>fct_part3_2</receiver>
   <slot>supprimerRes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ajouterRes()</slot>
  <slot>supprimerRes()</slot>
  <slot>ouvrirModifierRes()</slot>
 </slots>
</ui>
